## How much energy does it take to download a gigabyte of data over LTE?


It's not that uncommon to calcualte CO2 emissions associated with electricity by taking energy generated, the multiplying it my an emissions factor - usuallt grams of CO2 per kilowatt hour, to get a figure of CO2 emissions emitted as a result of using electricty.

I'm trying to work out the figures for 4G, and below represents me having a go, but I am in no way trained to do this.





### The only numbers I see in the units I want are from the 2012 Cloud begins with Coal Report

Here's the [link to the report](www.tech-pundit.com/wp-content/uploads/2013/07/Cloud_Begins_With_Coal.pdf)

> Reduced to personal terms, although charging up a single tablet or smart phone requires a negligible amount of electricity, using either to watch an hour of video weekly consumes annually more electricity in the remote networks than two new refrigerators use in a year. And as the world continues to electrify, migrating towards one refrigerator per household, it also evolves towards several smartphones and equivalent per person.

The workings are mentioned in the appendix, so I'm trying to see what these numbers might look like now.

> New refrigerator 350 kWh per EPA Energy Star; ~700 kWh/yr

> Weekly streaming HD from [network operations] + [network embodied energy]+ [tablet embodied energy];note, ignores data centers & end-­‐use tablet charging: ~ 300 kWh/yr

> wireless network operations from HD video 2.8 GB/hr per Netflix,network energy ~2kWh/GB. 

> Note energy use varies with location (type/age equipment), system utilization (see Auer et al, “How Much Energy is Needed to Run a Wireless Network?” June 2012).

> Network energy ranges from 19 kWh/GB The Mobile Economy, 2013, ATKearney, to ~2kWh/GB per CEET, The Power of Wireless Cloud, April 2013.

> Annualized embodied/manufacturing energy to produce tablet (details in this report) ~100 kWh/yr per tablet, and cell network operating energy equals annualized embodied energy of network equipment used for 5 years.

> Refrigerator embodied energy adds 5 -­‐10% to lifecycle energy use of refrigerator.

So his figures show 2KwH/GB. If we assume a figure of maybe 350g CO2 per Kwh (that's halfwat between European and US figures), this means we're looking at 700g of CO2 per Gigabyte.But what year is this based on?


### Let's take some figures from the 2012 power of wireless cloud report

Here's the [link to the CEET report](http://www.ceet.unimelb.edu.au/publications/ceet-white-paper-wireless-cloud.pdf)

This report is from the CEEE, and is artgues that public discussion needs to move away from the environmantal impact of data centres, and instead, to the impact of cellular wireless connectivity.

Here's the bit we care about with the numbers:

> The model first calculates the 4G LTE wireless connection between the user’s phone/tablet. Under typical circumstances, with 2010 technology, the energy consumption of a 4G LTE wireless access link ranges between 328 micro-Joules per bit and around 615 micro-Joules per bit [42]. The current annual energy efficiency improvement for wireless systems is approximately 26% [52]. Therefore, the 2010 values need to be adjusted to represent the expected energy per bit in 2015. This gives values of 73 micro-Joules per bit and 136 micro-Joules per bit. This paper adopted the lower value of 73 micro-Joules per bit

Got that?

In 2010:

- 328 micro-Joules per bit
- 615 micro-Joules per bit

And over 5 years, with 26% annual energy efficiency improvement, we get 

- In 2015, this is 73 micro-joules per bit as the lower range
- 2015 136 as the upper range

In the 


In [43]:
def annual_efficieny_gain(initial_val, no_of_years):
    val = initial_val
    for x in range(no_of_years):
        val *= 0.74
    return val


energy_per_bit_in_2015 = annual_efficieny_gain(328, 5)
energy_per_bit_in_2015

72.78341726719998

Okay, we're getting the same numbers as the report. Lets see assumed figures in 2018

In [37]:
energy_per_bit_in_2018 = annual_efficieny_gain(328, 8)
energy_per_bit_in_2018

29.493587478683846

Okay we now have a figure of… let's call it **29 micro-joules per bit**.

According to wikipedia, a joule is also known a watt-second.

A micro-joule is 10^-6 joules.

And a gigabyte is 10^9 bits.



In [44]:
energy_per_byte_in_2018 = energy_per_bit_in_2018 * 8

energy_per_megabyte_in_2018 = energy_per_byte_in_2018 * 1024

energy_per_gigabyte_in_2018 = energy_per_megabyte_in_2018 * 1024

# Actually, let's call it micro_joules_per_gigabyte_in_2018
micro_joules_per_gigabyte_in_2018 = energy_per_gigabyte_in_2018
micro_joules_per_gigabyte_in_2018

247410143.87238714

Okay now from microJoules to joules


In [39]:
joules_per_gigabyte = micro_joules_per_gigabyte_in_2018 / (1000 * 1000 )
joules_per_gigabyte

247.41014387238715

Huh?

247 joules per gigabyte is the same as 247 watt-seconds per gigabyte.

And now kilowatt-hours per gigabyte?

A kilowatt hour is 1000W, for an hour.



In [40]:
# how many joules in a kilowatt hour? Should be 3.6 million
1000 * (60 * 60)

3600000

In [41]:
kilowatt_hours_per_gigabyte = joules_per_gigabyte / (1000 * (60 * 60))
kilowatt_hours_per_gigabyte

6.872503996455199e-05